isis/scripts/downloadIsisData

```diff
@@ -123,17 +123,12 @@ def create_rclone_arguments(destination, mission_name, parsedArgs, rclone_kwargs
         set of kernels that will be downloaded
     """
     log.debug(f"Creating RClone command for {mission_name}")
-    mission_dir_name, source_type = mission_name.replace(":", "").split("_")
     
-    if (mission_dir_name == "legacybase"):
+    if (mission_name == "legacybase"):
         # We still want things to go into base
-        mission_dir_name = "base"
-        
-    log.debug(f"Mission_dir_name: {mission_dir_name}, source_type: {source_type}")
+        mission_name = "base"
```

Comment:
aforementioned legacybase download issue might actually be coming from here instead?